lib/ansible/modules/cloud/amazon/rds.py

```diff
@@ -1001,11 +1001,14 @@ def validate_parameters(required_vars, valid_vars, module):
 
     params = {}
     for (k, v) in optional_params.items():
-        if module.params.get(k) and k not in required_vars:
+        if module.params.get(k) in [True, False] and k not in required_vars:
```

Comment:
I think this should instead be `is not None`, since there are optional parameters that are neither true or false, such as `iops`, `size`, and `license_model`.